lib/home_page.dart

```diff
@@ -23,31 +30,65 @@ class HomePage extends StatelessWidget {
                   radius: Constants.radiusAvatar,
                 ),
               ),
-              Padding(
+              const Padding(
                 padding: EdgeInsets.all(Constants.padding),
                 child: Text(
                   Constants.name,
                   style: TextStyles.nameTextStyle,
                 ),
               ),
-              Padding(
+              const Padding(
                 padding: EdgeInsets.all(Constants.padding),
                 child: Text(
                   Constants.job,
                   style: TextStyles.jobTextStyle,
                 ),
               ),
-              Padding(
+              const Padding(
                 padding: EdgeInsets.all(Constants.padding),
                 child: Text(
                   Constants.mail,
                   style: TextStyles.mailTextStyle,
                 ),
               ),
+              Padding(
+                padding: const EdgeInsets.all(Constants.padding),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(
+                        left: Constants.paddingLeft,
+                      ),
+                      child: Text(
+                        '$_likesCounter',
+                        style: TextStyles.likeTextStyle,
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.only(
+                        right: Constants.paddingRight,
+                      ),
+                      child: IconButton(
+                        onPressed: _incrementLikes,
+                        icon: const Icon(Icons.thumb_up),
+                        color: Colors.green,
+                        iconSize: Constants.iconSize,
+                      ),
+                    )
+                  ],
+                ),
+              ),
             ],
           ),
         ),
       ),
     );
   }
+
+  _incrementLikes() {
+    setState(() {
+      _likesCounter++;
+    });
+  }
```

Comment:
Avoid defining method after the build(), please do it before

Reply:
done